plugins/nexus-repository-manager/src/api/nexus-repository-manager-api-client/nexus-repository-manager-api-client.ts

```diff
@@ -23,6 +23,19 @@ const NEXUS_REPOSITORY_MANAGER_CONFIG = {
   experimentalAnnotations: 'nexusRepositoryManager.experimentalAnnotations',
 } as const;
 
+/**
+ * Indicates that we want manifest v2 schema 2 if possible. It's faster
+ * for supporting servers to return and contains size information.
+ * @see @link{https://docs.docker.com/registry/spec/manifest-v2-2/#backward-compatibility|Backward compatibility}
+ */
+const DOCKER_MANIFEST_HEADERS = {
+  Accept: [
+    'application/vnd.docker.distribution.manifest.v2+json',
+    'application/vnd.docker.distribution.manifest.v1+json;q=0.9',
+    '*/*;q=0.8',
+  ].join(', '),
+};
```

Comment:
```suggestion
} as const satisfies HeadersInit;
```

When possible, try to use a const assertion for true constant variables.

Reply:
Got it, read up on the differences between `const x` and `...as const` and use of `satisfies`, thank you for the catch!